### DISCUSSION TITLE
I want use svg to define homepage-layout features icon

### DISCUSSION THREAD

**Discussion by JiaZhiheng** [vuejs/vitepress#1589] [Ideas]
I want use SVG to define homepage-layout features icon
but I have to use Emoji

**Comment by MarkusKeck** [comment]
I would also appreciate if the features in the home layout would support svg images, at the moment only emojis are supported.
Now the features are also clickable which is very handy, I think adding this feature would be a big improvement.

**Comment by kiaking** [comment]
Yeah we should add this feature. Please file an issue if it doesn't already exist 👀 (It might so please check.

**Comment by MarkusKeck** [comment]
@kiaking 
Filtered trough all the open issues and pr's and could not find anything regarding this topic.
I opened one: https://github.com/vuejs/vitepress/issues/1702
